purchase_packaging_default/models/purchase_order_line.py

```diff
@@ -12,7 +12,9 @@ class PurchaseOrderLine(models.Model):
     def _get_default_packaging(self):
         """From product get 1st packaging found ordered by sequence"""
         product_template = self.product_id.product_tmpl_id
-        return fields.first(product_template.packaging_ids)
+        return fields.first(
+            product_template.packaging_ids.filtered(lambda p: p.purchase)
+        )
```

Comment:
This approach is not optimal... You first loop over all the packaging and then return the first one.... 
```suggestion
        for pkg in  product_template.packaging_ids:
            if pkg.purchase:
                return pkg
        return product_template.packaging_ids.browse()
```